.circleci/config.yml

```diff
@@ -401,29 +410,23 @@ jobs:
           command: echo "Test modules succeeded"
 
   heartbeat_demo:
-    <<: *python_36_base
+    <<: *python_37_base
     steps:
       - checkout
-      - pip_install_no_dev
+      - build_test_docker
       - run:
           name: Run demo Ursula fleet, Alicia and the Doctor
-          command: ./scripts/demos/run_heartbeat_demo.sh
-      - store_artifacts:
-            path: /tmp/ursulas-logs
```

Comment:
Storing ursulas logs as CI artifacts has proven very useful many times to debug problems. Is there a way to extract logs and store them here as well?

Reply:
There might be.  Can someone tell me how do I look at the logs which have been "store_artifacts"ed?  So I can figure out how to implement this.

Reply:
Were using `appdirs` to determine the logging directory, on linux it at `~/.cache/nucypher/log/nucypher.log`